lib/fathead/mdn_css/parse.pl

```diff
@@ -298,24 +367,82 @@ sub parse_fragment_data {
                 my $dd  = $dt->next;
                 my $description;
                 if ($dd) {
-                    $description = build_abstract($dd->all_text);
+                    $description = build_abstract( $dd->all_text );
                 }
-                my @article_data = make_article($title, $description, $url);
+                my @article_data = make_article( $title, $description, $url );
                 write_to_file(@article_data);
             }
         }
     }
-    elsif ( $link =~ qr/time/ ) {
-        say "time $link";
-    }
-    elsif ( $link =~ qr/url/ ) {
-        say "url $link";
+    elsif ( $link =~ qr/time|frequency|angle/ ) {
+        my $h2 = $dom->at('h2#Summary');
+        my $ul_containing_fragments =
+          $h2->following->first( sub { $_->tag eq 'ul' } );
+        if ($ul_containing_fragments) {
+
+            for my $li ( $ul_containing_fragments->find('li')->each ) {
+                my $title = $li->at('a')->text;
+                my $url   = $link->clone->fragment($title);
+
+=begin
+              Taking 's' and 'ms' fragments for /time/ as examples
+              their current description appears as below:
+              s which represents a time in seconds...
+              ms which represents a time in milliseconds...
+
+              We remove the 'which' so that the description appears as:
+              s represents a time in seconds...
+              Which sounds right.
+=cut
+
+                my $paragraph = $li->all_text;
+                $paragraph =~ s/\s{1}which//;
+
+                #for /frequency/ and /angle/ the example code
+                #is contained in a table
+                #for /time/ it is contained in a <pre>
+                my $node_with_code = $link =~ /time/ ? 'pre' : 'table';
+                my $code_dom = $dom->at('h2#Examples')->following->first(
+                    sub {
+                        $_->tag eq $node_with_code;
+                    }
+                ) if $dom->at('h2#Examples');
+                my $code;
+                if ($code_dom) {
+                    if ( $code_dom->tag eq 'pre' ) {
+                        $code = $code_dom->all_text;
+                    }
+                    else {
+                        $code = $code_dom->at('tbody')->all_text;
```

Comment:
This may also be breaking the page, but it could be the content.

I noticed that `css ms` and `css contrast()` have broken snippets and break the page content slightly:

- https://moollaza.duckduckgo.com/?q=css+ms&ia=about
- https://moollaza.duckduckgo.com/?q=css+contrast()&ia=about